Mobile/starter_project_dartsmiths/lib/features/example/presentation/screen/profile.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+import '../widgets/aboutMe.dart';
+import '../widgets/myPosts.dart';
+import '../../data/datasources/appColor.dart';
+
+class Profile extends StatelessWidget {
+  const Profile({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: AppColor.background,
+      body: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(
+                top: 60,
+                left: 40,
+                right: 40,
+              ),
+              child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text(
+                      "Profile",
+                      style: TextStyle(
```

Comment:
[nit] I like that we are using global colors here

I would also recommend setting global text styles we have repeating text styles (like title texts, body texts, and their fonts...)

Reply:
You can use this as a template to define styles 
const TextStyle myTextStyle = TextStyle(
  fontSize: 16.0,
  fontWeight: FontWeight.bold,
  color: Colors.black,
);